.github/workflows/deno.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       matrix:
         # we test on both most recent stable version of deno (v1.x) as well as
         # the version of deno used by Run on Slack (v1.31.1)
-        deno-version: [v1.x, v1.31.1]
+        deno-version: [v1.x, v2.x]
```

Comment:
Do you want to update this comment? It mentions "as well as the version of deno used by Run on Slack (v1.31.1)` but we're now removing the `v1.31.1` reference.